pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
```

Comment:
Ah, ok - I'll remove it. See [this](https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma) though.